jujugui/static/gui/src/app/components/added-services-list/test-added-services-list.js

```diff
@@ -103,35 +103,52 @@ describe('AddedServicesList', () => {
     assert.deepEqual(output, expected);
   });
 
-  it('performs the necessary work to focus a service', () => {
-    var allServices = [{
-      get: () => 'trusty/wordpress',
-      set: sinon.stub(),
-      setAttrs: sinon.stub()
-    }, {
-      get: () => 'trusty/apache',
-      set: sinon.stub(),
-      setAttrs: sinon.stub()
-    }, {
-      get: () => 'trusty/mysql',
-      set: sinon.stub(),
-      setAttrs: sinon.stub()
-    }];
-    var services = {
+  function createService(attrs) {
+    return {
+      attrs: attrs || {},
+
+      get: function(key) {
+        return this.attrs[key];
+      },
+
+      set: function(key, value) {
+        this.attrs[key] = value;
+      },
+
+      setAttrs: function(attrs) {
+        for (var key in attrs) {
+          this.attrs[key] = attrs[key];
+        }
+      }
+    };
+  }
+
+  function createModelList(models){
+    return {
       each: (cb) => {
-        allServices.forEach(cb);
+        models.forEach(cb);
```

Comment:
I called them "models" because that's what they are, architecturally - objects that model and provide access to our datastore. It has nothing to do (and should have nothing to do) with whatever terminology is used in the UI.